Hw18 webflux chat room1408

uniCoder05 opened the pull request:
webflux-chat
- в чат из примера к вебинару 38 добавлена специальная комната: 1408;
- в этой комнате нельзя отправлять сообщения, но в нее приходят все сообщения из всех других комнат;
- при входе в комнату 1408 загружаются все сообщения из всех комнат (по аналогии с типовыми комнатами).

Commit:
Homework 18 webflux-chat room1408
init

Commit:
Homework 18 room1408
webflux-chat
- в чат из примера к вебинару 38 добавлена специальная комната: 1408;
- в этой комнате нельзя отправлять сообщения, но в нее приходят все сообщения из всех других комнат;
- при входе в комнату 1408 загружаются все сообщения из всех комнат (по аналогии с типовыми комнатами).

petrelevich commented on hw18-webflux-chat-room1408/datastore-service/src/main/java/ru/petrelevich/repository/MessageRepository.java:
```diff
@@ -0,0 +1,19 @@
+    Flux<Message> findByRoomId(@Param("roomId") String roomId);
+
+    @Override
+    @Query("select * from message")
```
стоит добавить сортировку, например по идентификатору сообщений, чтобы вывод был консистентный.

uniCoder05 replied:
исправлено - сортировка по id добавлена

Commit:
Homework 18 room1408
fixed
в запросе на получение всех сообщений добавлена сортировка по id
@Query("select * from message order by id")
